graylog2-server/src/test/resources/org/graylog2/security/ldap/base.ldif

```diff
@@ -35,6 +35,12 @@ objectClass: top
 cn: Engineers
 uniqueMember: cn=John Doe,ou=users,dc=example,dc=com
 
+dn: cn=Whitespace Engineers,ou=groups,dc=example,dc=com
```

Comment:
Whitespace Engineers – Engineering something you can't see since 1942 (also non-breaking). 😆